src/python/chevahbs

```diff
@@ -201,6 +197,13 @@ chevahbs_install() {
             execute $MAKE install DESTDIR=$INSTALL_FOLDER
             ;;
     esac
+
+    case $OS in
+        # For now we only install PIP on ArchLinux
+        "archlinux")
+            execute $INSTALL_FOLDER/bin/python ../../python-modules/get-pip.py $PIP_ARGS
+            ;;
+    esac
```

Comment:
I was hoping to see this moved outside of the python build.

This script should be used to build a python as closed to upstream as possible...

then in chevah_build we can do all kind of customization, like installing pip and extra libs